smart_open/gcs.py

```diff
@@ -13,7 +14,7 @@
 
 import google.cloud.exceptions
 import google.cloud.storage
-import google.auth.transport.requests as google_requests
+import google.auth.transport.requests
```

Comment:
We need to update https://github.com/RaRe-Technologies/smart_open/blob/22761352d1013ed4340eec6c592f35eaadf2ed04/smart_open/tests/test_gcs.py#L463 to match this for mocks to work:

```
'google.auth.transport.requests',
```